ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -2865,6 +2865,17 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-vsphere-ipi-ovn-shared-to-local-gateway-mode-migration
+- as: vsphere-ipi-ovn-winc-p2-f14
+  cron: 32 12 8,23 * *
+  remote_api: true
```

Comment:
remote_api is not needed now, as the default value has changed to true since June, 2023

Reply:
when I remove this it leads to a bunch of files modified , will ping on slack .. do we need to do it?

Reply:
OK. Let's leave it as it is now.  I'll use a follow up PR to remove them in batch.